.azure-devops/nightly.yml

```diff
@@ -62,37 +87,44 @@ stages:
 
       - template: templates/install-and-record-version.yml
 
-  - stage: 'test_edge'
-    displayName: 'Run all tests with edge CLI'
+  - stage: 'test_min'
+    displayName: 'Run all tests with min CLI'
     pool:
       vmImage: ${{ parameters.vmImage }}
     dependsOn: build
     jobs:
     - template: templates/nightly-tests.yml
       parameters:
-        azureCLIVersion: edge
+        azureCLIVersion: min
         pythonVersion: $(pythonVersion)
-  
-  - stage: 'test_min'
-    displayName: 'Run all tests with min CLI'
+        testADT: ${{ parameters.testADT }}
+        testCentral: ${{ parameters.testCentral }}
+        testDPS: ${{ parameters.testDPS }}
+        testHub: ${{ parameters.testHub }}
+
+  - stage: 'test_msi'
+    displayName: 'Run all tests against Windows MSI install'
     pool:
-      vmImage: ${{ parameters.vmImage }}
-    dependsOn: test_edge
+      vmImage: 'vs2017-win2016'
+    dependsOn: test_min
```

Comment:
Should the MSI environment tests necessarily dependOn `test_min` completion?

Reply:
Not necessarily, it's just to avoid them running simultaneously - which may be less of an issue now that we have the scaling / service connection issues worked out. Do you see an issue running the nightly suites simultaneously? Would be a nice boost in performance.

Reply:
Actually yea....lets hold off on simultaneous execution due to the load of provisioned iothubs per iot hub test job. This is something we can tackle in a future iteration once we go through some potential optimizations to test parallelization.